### PR TITLE
PP-3914: extract container mass

### PR DESCRIPTION
**Pull request by joshhacksthings**
Update harness to extract mass from `refs`.

### CODE REVIEW

**Review by HugoMayoral — DISMISSED** [review]
LGTM!

File: autoprotocol/container.py [hunk 1]
```diff
@@ -115,6 +118,37 @@ def add_properties(self, properties):
...
+        Returns
+        -------
+        Well
+            Well with modified mass
+
+        Raises
+        ------
+        TypeError
+            Incorrect input-type given
+        ValueError
```
Comment (xiliuStrateos) [inline autoprotocol/container.py:139]: This method does not raise `ValueError`

File: autoprotocol/container.py [hunk 2]
```diff
@@ -115,6 +118,37 @@ def add_properties(self, properties):
...
         return self
 
+    def set_mass(self, mass):
+        """
+        Set the theoretical mass of contents in a Well.
+
+        Parameters
+        ----------
+        mass : str, Unit
+            Theoretical volume to indicate for a Well.
```
Comment (xiliuStrateos) [inline autoprotocol/container.py:128]: volume or mass?

**Review by xiliuStrateos — CHANGES_REQUESTED** [review]
nit - doc str change

**Review by xiliuStrateos — APPROVED** [review]

**Pull request merged**